addons/mail/static/src/scss/variables/primary_variables.scss

```diff
@@ -2,7 +2,7 @@ $o-mail-Avatar-size: 42px !default;
 $o-mail-ChatWindow-width: 360px !default; // same value as CHAT_WINDOW_WIDTH
 $o-mail-Chatter-mobile-gap: 2% !default;
 $o-mail-sidebar-icon-opacity: 0.7 !default;
-$o-mail-Discuss-inspector: 250px !default;
+$o-mail-Discuss-inspector: 300px !default;
```

Comment:
Note that pinned message panel takes 425px (it uses `flex-basis`, and it has precedence over `width`).
I'm not entirely sure we want to keep different sizes for inspectors. If that's the case, would be nice to define it in action

Reply:
I was wondering the same but I feel 425 seems too big for others that are more permanently open (eg. member list), but less than 425 is too small for pinned messages.

Reply:
Could be worth having 2 official sizes for inspector